Framework/DataHandling/src/LoadNexusLogs.cpp

```diff
@@ -772,7 +773,8 @@ void LoadNexusLogs::loadLogs(::NeXus::File &file, const std::string &absolute_en
           if (!block_list.empty()) {
             bool skip = false;
             for (const auto &block : block_list) {
-              if ((*it).substr((*it).find_last_of("/") + 1) == block) {
+              Poco::Glob glob(block);
```

Comment:
Actually there are a lot of the instances using `poco::glob` directly instead of going through kernel.
I think it would make more sense to bring this up in the next technical meeting to ask if people would prefer using poco::glob directly or via a wrapper from kernel.
Once a decision is made, we can go ahead and implement the changes (either remove the wrapper, or force all globing via the wrapper).
This way we will have a much better change to have consistent way to perform globing within Mantid.